x-pack/plugins/actions/server/builtin_action_types/webhook.ts

```diff
@@ -93,17 +103,19 @@ export async function executor(
 ): Promise<ActionTypeExecutorResult> {
   const actionId = execOptions.actionId;
   const { method, url, headers = {} } = execOptions.config as ActionTypeConfigType;
-  const { user: username, password } = execOptions.secrets as ActionTypeSecretsType;
   const { body: data } = execOptions.params as ActionParamsType;
 
+  const secrets: ActionTypeSecretsType = execOptions.secrets as ActionTypeSecretsType;
+  const basicAuth =
+    secrets.user && secrets.password
```

Comment:
So, just thinking a test like `secrets.user` / `secrets.password` will "fail" when the string is the empty string.  Wondering when the last time I used a service that allowed/required an empty string for either.  I think it's been a long time ...

```console
> console.log(`empty string is ${'' ? true : false}`)
empty string is false
```

Reply:
OK, I can replace it with a lodash `isString` or something equivalent